tox.ini

```diff
@@ -12,6 +12,7 @@ deps =
     twlatest: Twisted[tls]
     twtrunk: https://github.com/twisted/twisted/archive/trunk.zip#egg=Twisted[tls]
     twlowest: Twisted[tls]==15.5.0
+    git+https://github.com/certbot/certbot.git@master#egg=acme&subdirectory=acme
```

Comment:
I'm not planning to land this branch at all until there's an actual upstream release with this functionality, so the URL here is strictly a temporary measure. I probably should have used a particular revision ID instead of just `master`, I'll try remember that for next time I do a hack like this.